### ISSUE TITLE
[A11y]: SVGs need alternatives for screenreaders

### ISSUE THREAD

**Issue by codewordcreative** [passkeydeveloper/whatarepasskeys.info#48]
### Page URL / Component

https://whatarepasskeys.info

### Barrier Category

Screen Reader Incompatibility / Missing Labels

### Assistive Technology Used

N/A

### Detailed Description

All the SVGs in the comparison table where there is an X, - etc. need a non-visual alternative description to match partial, full, etc.
